node/src/components/chain_synchronizer/metrics.rs

```diff
@@ -80,14 +74,6 @@ impl Metrics {
             "chain_sync_total_duration_seconds",
             "total time in seconds of syncing the chain",
         )?;
-        let chain_sync_emergency_restart_duration_seconds = IntGauge::new(
-            "chain_sync_emergency_restart_duration_seconds",
-            "time in seconds of handling the emergency restart",
-        )?;
-        let chain_sync_upgrade_duration_seconds = IntGauge::new(
-            "chain_sync_upgrade_duration_seconds",
-            "time in seconds of handling the upgrade",
-        )?;
```

Comment:
Why do we not need these two metrics anymore? What changed when movnig the archival sync to background task?